File: 349-intersection-of-two-arrays/intersection-of-two-arrays.js
/**
 * @param {number[]} nums1
 * @param {number[]} nums2
 * @return {number[]}
 */
var intersection = function(nums1, nums2) {
    const seen = new Set(nums1)
    const result = new Set();
    for(const num of nums2){
        if (seen.has(num))
            result.add(num)
    }
    return Array.from(result)
};

File: 349-intersection-of-two-arrays/intersection-of-two-arrays.py
class Solution(object):
    def intersection(self, nums1, nums2):
        nums3 = []
        for i in range(len(nums1)):
            for j in range(len(nums2)):
                if nums1[i] == nums2[j] and nums1[i] not in nums3:
                    nums3.append(nums1[i])
        return nums3